pkg/config/s3.go

```diff
@@ -2,14 +2,14 @@ package config
 
 // S3Config specifies the properties required to use S3 as the storage backend
 type S3Config struct {
-	Region                             string `validate:"required" envconfig:"AWS_REGION"`
-	Key                                string `envconfig:"AWS_ACCESS_KEY_ID"`
-	Secret                             string `envconfig:"AWS_SECRET_ACCESS_KEY"`
-	Token                              string `envconfig:"AWS_SESSION_TOKEN"`
-	Bucket                             string `validate:"required" envconfig:"ATHENS_S3_BUCKET_NAME"`
-	UseDefaultConfiguration            bool   `envconfig:"AWS_USE_DEFAULT_CONFIGURATION"`
-	ForcePathStyle                     bool   `envconfig:"AWS_FORCE_PATH_STYLE"`
-	CredentialsEndpoint                string `envconfig:"AWS_CREDENTIALS_ENDPOINT"`
-	AwsContainerCredentialsRelativeURI string `envconfig:"AWS_CONTAINER_CREDENTIALS_RELATIVE_URI"`
-	Endpoint                           string `envconfig:"AWS_ENDPOINT"`
+	Region                             string `validate:"required" envconfig:"AWS_REGION" json:"region"`
+	Key                                string `envconfig:"AWS_ACCESS_KEY_ID" json:"key"`
+	Secret                             string `envconfig:"AWS_SECRET_ACCESS_KEY" json:"secret"`
```

Comment:
@fedepaol I don't think this config value should be returned. And the same for other secrets. Maybe have a struct tag that indicates it should be hidden, and then read that when you encode it for JSON?

Reply:
As part of the discussion in [https://github.com/gomods/athens/issues/1503#issuecomment-604677096](https://github.com/gomods/athens/issues/1503#issuecomment-604677096) we agreed that it was fine to expose (also) the sensitive data on a different port, as the endpoint was reachable only with direct machine / container access.